x-pack/plugins/monitoring/server/config.test.ts

```diff
@@ -115,6 +103,22 @@ describe('config schema', () => {
 });
 
 describe('createConfig()', () => {
+  const MOCKED_PATHS = [
```

Comment:
@spalger Is this change okay? 

I got odd failures when this block was higher up, it was passing `jest-worker/index.js` to this mock which I guess was due to Prettier being run as part of the inline snapshot generation?

Reply:
I don't think there's any particular issue here, but I also don't understand what you're describing as the issue it's preventing. Jest is passing a worker to the array??

Reply:
So before, when this was higher up, this hook was run for the test above which has a `toMatchInlineSnapshot`. 

Now somehow when that snapshot is updated, it gets passed to Prettier since Jest detects that we use Prettier and somewhere in that chain something calls fs.readFileSync but since that is mocked it ends up in this mock which throws an errors 🤷🏼

Reply:
Ahh, is the mock below which you're referring to. Yeah, mocking the filesystem is tricky, I have sometimes used `jest.requireActual('fs')` to pass through unexpected filenames to node to fix this.

Reply:
But moving this down so it won't run for snapshot test fixes it, but I just got nervous due to how the mock is dealing with the paths to our certs, don't wanna leak that into build logs etc. :) Which I assume is the goal of the mock.

Reply:
Ah, no, those certs are [committed to the repo](https://github.com/elastic/kibana/tree/main/packages/kbn-dev-utils/certs). That's just done to make the snapshots more legible and resistant to changes when the certs are rotated.

Reply:
Oh I see, great! No need to worry than, thanks for the double check! 🙏🏼